test/framework/executor_itf/executor_itf.hpp

```diff
@@ -0,0 +1,315 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_TEST_FRAMEWORK_EXECUTOR_ITF_HPP
+#define IROHA_TEST_FRAMEWORK_EXECUTOR_ITF_HPP
+
+#include <memory>
+
+#include <boost/optional.hpp>
+#include "ametsuchi/command_executor.hpp"
+#include "ametsuchi/impl/postgres_options.hpp"
+#include "ametsuchi/query_executor.hpp"
+#include "ametsuchi/tx_executor.hpp"
+#include "common/result.hpp"
+#include "framework/common_constants.hpp"
+#include "framework/executor_itf/executor_itf_helper.hpp"
+#include "interfaces/commands/command.hpp"
+#include "interfaces/queries/query.hpp"
+#include "logger/logger_fwd.hpp"
+#include "logger/logger_manager_fwd.hpp"
+#include "module/shared_model/command_mocks.hpp"
+#include "module/shared_model/query_mocks.hpp"
+
+namespace shared_model {
+  namespace crypto {
+    class PublicKey;
+  }
+  namespace interface {
+    class MockCommandFactory;
+    class MockQueryFactory;
+    class Transaction;
+  }
+}  // namespace shared_model
+
+namespace iroha {
+  namespace integration_framework {
+
+    class ExecutorItf {
+     public:
+      /**
+       * Create an ExecutorItf.
```

Comment:
IMO the previous version of doc was better, since it listed that admin account is created and db connection is initialized.